Update docs related to styles (#62)
asyncapi/asyncapi-react: docs/configuration/style-modification.md

# Style Modification

## Overview

> This document describes how you can apply the styles for the AsyncApi component.

Reviewer: ```suggestion
This document describes how you can apply the styles for the AsyncAPI component.
```

## Changing styles

> In the AsyncApi component we use pure css styling. Each component inside root component has a unique css class. Each class has form: `asyncapi__{ELEMENT}--{MODIFIER}`, where:

Reviewer 2: maybe `each html tag has unique css class`? from css perspective there are html tags/elements, not components

> In the AsyncApi component we use pure css styling. Each component inside root component has a unique css class. Each class has form: `asyncapi__{ELEMENT}--{MODIFIER}`, where:

Reviewer: ```suggestion
In the AsyncApi component, we use pure CSS styling. Each HTML tag inside the root component has a unique CSS class in the `asyncapi__{ELEMENT}--{MODIFIER}` format, where:
```

> - `{ELEMENT}` - is a name of specific element. Each element name is the concatenation of the names of the elements in which it is located. For example: `asyncapi__channel-header-title` is located in `header` HTML element of `channel` element. 

Reviewer 2: ```is located in `header` HTML element``` - is this always true? This may lead to situation in which someone does something like `header.asyncapi__channel-header-title {}`, but in reality there could be div element. Maybe instead of outlining things like that just say we follow BEM and link to it?

Author: But here I described that `title` is in `header`, not `*-title` is HTML header.

> - `{ELEMENT}` - is a name of specific element. Each element name is the concatenation of the names of the elements in which it is located. For example: `asyncapi__channel-header-title` is located in `header` HTML element of `channel` element. 

Reviewer: ```suggestion
- `{ELEMENT}` is the name of a specific element. Each element name is the concatenation of the names of the elements in which it is located. For example, `asyncapi__channel-header-title` is located in the `header` HTML element of the `channel` element. 
```

> - `{MODIFIER}` - is a modifier for `{ELEMENT}`. Very few elements have a modifier. This is usually badge, button and similar, generic components.

Reviewer: ```suggestion
- `{MODIFIER}` is a modifier for `{ELEMENT}`. Very few elements have a modifier. This is usually a badge, button, or similar, generic components.
```

> For changing styles, you must create (or if you use [default](../../library/src/styles/fiori.css) style, modify) appropriate class.

Reviewer: ```suggestion
To change styles, create an appropriate class or modify it if you use the [default](../../library/src/styles/fiori.css) style.
```

> > **NOTE**: We recommend copy default styles from [here](../../library/src/styles/fiori.css) and changing them at its sole discretion.

Reviewer 2: **copying**

and did you really mean _discretion_?

> > **NOTE**: We recommend copy default styles from [here](../../library/src/styles/fiori.css) and changing them at its sole discretion.

Reviewer: ```suggestion
> **NOTE**: We recommend that you first [copy](../../library/src/styles/fiori.css) the default styles to a separate file and then modify them as you prefer.
```
